### PR TITLE
Import data generators from CAP

### PR DESCRIPTION
**Pull request by ActionAnalytics**
Data generates with 

```./generate.sh 10 Static Saved```

Screen Shot 2020-02-28 at 1 02 38 PM
Screen Shot 2020-02-28 at 1 03 01 PM

### CODE REVIEW

**Comment by ActionAnalytics** [comment]
Up to date with Dylan's changes:
Screen Shot 2020-02-28 at 5 52 54 PM
Screen Shot 2020-02-28 at 5 56 10 PM
Screen Shot 2020-02-28 at 5 57 08 PM

**Comment by ActionAnalytics** [comment]
Screen Shot 2020-03-01 at 12 46 00 AM

**Comment by ActionAnalytics** [comment]
Added Privacy and Project file cleanliness Guardrails because it's needed.
Still passing all tests:
Screen Shot 2020-03-01 at 7 03 18 PM
Screen Shot 2020-03-01 at 7 03 40 PM

File: api/test/factories/inspection_factory.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    , name             : author.fullName
+    , label            : faker.lorem.sentence()
+    , case             : faker.lorem.sentence()
+    , email            : author.emailAddress
+    , startDate        : startDate
+    , endDate          : endDate
+    , elements         : [factory_helper.ObjectId(), factory_helper.ObjectId(), factory_helper.ObjectId()]
+    , customProjectName: ''
+    , project          : factory_helper.ObjectId()
+    , inspectionId     : "abc123-abc123-abc123"  // TODO: mock this out properly
```
Comment (KitArmstrong) [inline api/test/factories/inspection_factory.js]: Does this still need to be done?

Comment (ActionAnalytics) [reply api/test/factories/inspection_factory.js]: Yes that was one to ask the author of that file.  Not in this revision though

Comment (ActionAnalytics) [reply api/test/factories/inspection_factory.js]: @KitArmstrong captured it in ticket EE-870, thanks for flagging that

**Pull request merged**